src/index.js

```diff
@@ -18,7 +18,7 @@ const CID_CBOR_TAG = 42
  * @returns {cborg.Token[]|null}
  */
 function cidEncoder (obj) {
-  if (obj.asCID !== obj) {
+  if (obj['/'] !== obj.bytes) {
```

Comment:
this isn't accounting for use of multiformats versions less than the one you just released, we probably need both checks in there for now

```suggestion
  if (obj.asCID !== obj  && obj['/'] !== obj.bytes) {
```